src/main/java/core/basesyntax/FindNumbers.java

```diff
@@ -11,6 +11,17 @@
      * числа 92, 18, 26 и 0.</p>
      */
     public int[] getAllNumbers(String text) {
-        return null;
+        String[] temp = text.replaceAll("[^\\d\\-]+", " ")
+                .trim().split(" ");
+
+        int[] result = new int[temp.length];
+        int counter = 0;
+        for (String s : temp) {
+            if (!s.equals("")) {
+                result[counter++] = Integer.parseInt(s) * 2;
+            }
+        }
```

Comment:
let's use another loop here
we should be able to remove this counter and `if` check